### PR TITLE
Add String.last

### PR DESCRIPTION
**Pull request by Dilven**

### CODE REVIEW

File: __tests__/String/last.test.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { S, O } from '../..'
+
+describe('last', () => {
+  it('returns None if the given string is empty.', () => {
+    expect(S.last('')).toEqual(O.None)
+  })
+  it('returns Some(value) where value is the last character of the string', () => {
+    expect(S.last('last-test')).toEqual('t')
+  })
+})
```
Comment (mobily) [inline __tests__/String/last.test.ts:10]: could you please add a test with the `*` string as a first argument of `it`? (it automatically adds an example in the documentation)

File: src/String/String.res [hunk 2]
```diff
@@ -16,6 +16,14 @@ export concat = (str, append) => Js.String2.concat(str, append)
 )
 export head = str => Js.String2.slice(str, ~from=0, ~to_=1)
 
+%comment(
+  "Returns `Some(value)` where `value` is the last character of the string, or `None` if the given string is empty."
+)
+export last = str => {
+  let lastChar = Js.String2.sliceToEnd(str, ~from=length(str) - 1)
+  lastChar == "" ? None : Some(lastChar)
+}
```
Comment (mobily) [inline src/String/String.res:25]: I'm thinking, that we could reuse the `get` function instead :) 
```rescript
export last = str => get(str, length(str) - 1)
```

File: src/String/String.res [hunk 3]
```diff
@@ -16,6 +16,10 @@ export concat = (str, append) => Js.String2.concat(str, append)
 )
 export head = str => Js.String2.slice(str, ~from=0, ~to_=1)
 
+%comment(
+  "Returns `Some(value)` where `value` is the last character of the string, or `None` if the given string is empty."
+)
+
```
Comment (mobily) [inline src/String/String.res:22]: seems like this comment should be moved to the line before the `last` function :)

Comment (Dilven) [reply src/String/String.res]: oh you are right!

**Review by mobily — APPROVED** [review]
thank you again! 💯

**Pull request merged**